dockerfiles/verify_packages/centos/install.sh

```diff
@@ -17,7 +17,8 @@ yum --enablerepo=remi-php${PHP_MAJOR}${PHP_MINOR} install -y \
     php-fpm \
     php-opcache \
     php-pear \
-    mod_php
+    mod_php \
+    $([ ${PHP_MAJOR} == "5" ] && [ ${PHP_MINOR} -lt 6 ] && echo "php-json")
```

Comment:
The verification tests are still failing in CI after this change. Are the correct container images pushed up? If so, I'm assuming the json extension is being loaded as a shared lib in this case.

In either case, we'll need to do the same [`dlsym()` trick we used for curl](https://github.com/DataDog/dd-trace-php/blob/e8271f0/ext/php8/handlers_curl.c#L332-L359) to grab the missing symbol addresses.